Tests/SentryTests/Integrations/SentryUIPerformanceTrackerTests.swift

```diff
@@ -40,13 +45,28 @@ class SentryUIPerformanceTrackerTests: XCTestCase {
         super.setUp()
         fixture = Fixture()
     }
+
+    func testSwizzlingCall() {
```

Comment:
`m`: Can we add a comment that we call these methods to make sure the swizzling of them calls the original method, please? At first glance, I was wondering why there is no assertion in the test.